tests/e2e/lib.sh

```diff
@@ -331,6 +335,40 @@ setup_podsecuritypolicies_config() {
     fi
 }
 
+# wait_for_collectors_to_be_operational() ensures that collector pods are able
+# to load kernel objects and create network connections.
+wait_for_collectors_to_be_operational() {
+    info "Will wait for collectors to reach a ready state"
+
+    local readiness_indicator="Successfully established GRPC stream for signals"
+    local start_time
+    start_time="$(date '+%s')"
+    local all_ready="false"
+    while [[ "$all_ready" == "false" ]]; do
+        all_ready="true"
+        for pod in $(kubectl -n stackrox get pods -l app=collector -o json | jq -r '.items[].metadata.name'); do
+            echo "Checking readiness of $pod"
+            if kubectl -n stackrox logs -c collector "$pod" | grep "$readiness_indicator" > /dev/null 2>&1; then
+                echo "ready"
```

Comment:
This output is a bit useless, could you invert the condition (and pull up the else branch up)?

Reply:
> This output is a bit useless, could you invert the condition (and pull up the else branch up)?

I'm not a fan of weird inverted grep syntax for readability and I've tested this path so I'd hate to switch it around at this point :( I'd still echo something in the ready state.